### PR TITLE
feat: add translations for section 51 static content

### PR DESCRIPTION
**Pull request by cprieststephens**
## Describe your changes

[APPLICS 88](https://pins-ds.atlassian.net/jira/software/c/projects/APPLICS/boards/242?selectedIssue=APPLICS-88): 
- Add Welsh translations for section 51 static content
- Update conditional logic with language parameters so correct content continues to be displayed once a language is selected

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [x] I have included unit tests to cover any testable code changes

### CODE REVIEW

**Review by emil-placheta — DISMISSED** [review]
lgtm

File: packages/forms-web-app/src/pages/projects/section-51/index/view.njk [hunk 1]
```diff
@@ -27,48 +27,49 @@
 				</span>
 
 				<h1 class="govuk-heading-xl">
-					{{ title }}
+					{{ t('section51Index.heading1') }}
 				</h1>
 
-				{% if advice.length > 0 or(advice.length == 0 and queryUrl != '') %}
-					<p class="govuk-body">The list below includes a record of advice we have provided for this project.</p>
+				{% if advice.length > 0 or (advice.length == 0 and (queryUrl != '?lang=en' and queryUrl != '?lang=cy' and queryUrl != '')) %}
```
Comment (emil-placheta) [inline packages/forms-web-app/src/pages/projects/section-51/index/view.njk:33]: This is not very readable. Maybe it's worth having something like `adviceExists` and `showNoAdviceMwssage` etc returned from the controller/view-model to deal with the conditional rendering of those elements?

File: packages/forms-web-app/src/pages/projects/section-51/index/view.njk [hunk 2]
```diff
@@ -27,48 +27,49 @@
...
 
 				<p class="govuk-body">
-					There is a statutory duty, under <a class="govuk-link" href="https://www.legislation.gov.uk/ukpga/2008/29/section/51">section 51 of the Planning Act 2008</a>, around an application or potential application.
-					This includes recording the name of the person who requested advice and the advice given.
-					This information has to be made publicly available.
+					{{ t('section51Index.paragraph2', { link: '<a class="govuk-link" href="' + "https://www.legislation.gov.uk/ukpga/2008/29/section/51" + '">' + t('section51Index.paragraph2LinkText1') + '</a>' }) | safe }}
 				</p>
 
-				{% if advice.length > 0 or queryUrl != '' %}
+				{% if advice.length > 0 or (queryUrl != '?lang=en' and queryUrl != '?lang=cy' and queryUrl != '') %}
```
Comment (emil-placheta) [inline packages/forms-web-app/src/pages/projects/section-51/index/view.njk:47]: similarly to above

File: packages/forms-web-app/src/pages/projects/section-51/index/view.njk [hunk 3]
```diff
@@ -27,48 +27,49 @@
...
 					</p>
 
-					{{ uiSearchBar('section-51-page-search-bar', 'searchTerm', searchTerm, 'Search by key words or the name of person the advice was given to.') }}
+					{{ uiSearchBar('section-51-page-search-bar', 'searchTerm', searchTerm, t('section51Index.paragraph3'), t('common.search') ) }}
 				{% endif %}
 
-				{% if advice.length == 0 and queryUrl != '' %}
-					<p class="govuk-body">No advice was found matching your search term.</p>
-					<p class="govuk-body">Would you like to clear your search to view all available advice instead?</p>
+				{% if advice.length == 0 and (queryUrl != '?lang=en' and queryUrl != '?lang=cy' and queryUrl != '') %}
```
Comment (emil-placheta) [inline packages/forms-web-app/src/pages/projects/section-51/index/view.njk:59]: and here

**Review by emil-placheta — CHANGES_REQUESTED** [review]
We should try to simplify those if statements and move that logic out of the template. It will be easier to handle in the controller/view-model and make template easier to read/understand.

File: packages/forms-web-app/src/pages/projects/section-51/index/_utils/advice-helpers.js [hunk 4]
```diff
@@ -45,8 +45,13 @@ const doesAdviceExist = (advice) => {
 	return advice.length > 0;
 };
 
+// doesAdviceQueryUrlExist checks whether the user has attempted a search within the available advice for a case.
+// If queryUrl does not contain 'search', no search has been attempted meaning no advice is available for the case.
+// If queryUrl contains 'search', a search has been attempted meaning advice is available.
+// Used in conjunction with doesAdviceExist, this function clarifies whether no advice has been returned because there is none available for the case or because no advice matches the entered search term.
```
[the remark this answers is not in the document]
Comment (cprieststephens) [reply packages/forms-web-app/src/pages/projects/section-51/index/_utils/advice-helpers.js]: Much better!

File: packages/forms-web-app/src/pages/projects/section-51/index/view.njk [hunk 5]
```diff
@@ -10,7 +10,8 @@
 {% from "components/wrappers/related-guides-wrapper.njk" import relatedGuidesWrapper %}
 {% from "projects/section-51/index/_components/results.njk" import sectionResults %}
 
-{% set pageTitle = title %}
+{% set pageTitle = t('section51Index.heading1') %}
+{% set noAdviceExists = adviceExists == false %}
```
Comment (emil-placheta) [inline packages/forms-web-app/src/pages/projects/section-51/index/view.njk:14]: Sorry, missed that last time, you don't need to set `noAdviceExists`.
If you want to check for advice _not_ existing in a conditional statement, you can negate it in nunjucks like you would in javascript, but using different syntax:

`{% if(not adviceExists) %}
....
{% endif %}`

File: packages/forms-web-app/src/pages/projects/section-51/index/view.njk [hunk 6]
```diff
@@ -27,53 +28,54 @@
 				</span>
 
 				<h1 class="govuk-heading-xl">
-					{{ title }}
+					{{ t('section51Index.heading1') }}
 				</h1>
 
-				{% if advice.length > 0 or(advice.length == 0 and queryUrl != '') %}
-					<p class="govuk-body">The list below includes a record of advice we have provided for this project.</p>
+				{% if adviceExists or (noAdviceExists and adviceSearchResult) %}
```
Comment (emil-placheta) [inline packages/forms-web-app/src/pages/projects/section-51/index/view.njk:34]: `noAdviceExists` could be `not noAdviceExists`

**Review by emil-placheta — CHANGES_REQUESTED** [review]

**Review by emil-placheta — APPROVED** [review]

**Pull request merged**